.travis.yml

```diff
@@ -6,7 +6,7 @@ addons:
     - python-scipy
 install:
   - pip install flake8  # Not yet available in Ubuntu precise (https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise)
-  - pip install -r requirements.txt
```

Comment:
Sorry @fpagnoux I don't have time to understand why but removing requirements works, and I don't see in which case OpenFisca gets installed with pip install -r.